README.md

```diff
@@ -5,46 +5,52 @@ awesome-mycroft-community
+The primary criterion for inclusion is simple: Projects listed here are large undertakings, with committed developers who can engage with users. This does not mean that projects must be in a usable state, especially if those projects are meant to attract contributors! Everything is in early development at some point.
 
 (Contributors: Please submit all your maintained projects. Please don't submit abandonware. We all have a lot of awesome repos we haven't touched since 2019. Let's not list those.)
 
-- [Device Images/Builds](#device-images/builds)
-- [STT/TTS replacements and utilities](#STT/TTS-replacements-and-utilities)
-  - [STT](##STT)
-  - [TTS](##TTS)
-- [Wake word listeners and utilities](#wake-word-listeners-and-utilities)
-- [Skill management](#skill-management)
-- [GUI](#GUI)
-- [HiveMind](#HiveMind)
-- [Mobile](#Mobile)
-- [Other Thin Clients](#Other-Thin-Clients)
-- [System Integrations](#System-Integrations)
+----------------------------------------------------------------------------------------------------------------------------------------------
 
-# Device Images/Builds
+### Distributions, Images, Projects Shipping Mycroft
+**_A list of Products, Projects, Distributions and Images shipping Mycroft with special emphasis on Voice Input, Voice Assistance targeting specific hardware like SBC's, Smart Speakers, Home Entertainment Systems_**
 
-# STT/TTS replacements and utilities
-## STT
-### Systems
-### Utilities
-## TTS
-### Systems
-### Utilities
+- **Open Voice OS**: Open Voice OS is a community powered Linux distribution purpose-built to showcase the power of Open Source Voice AI for a range of devices. [Project Link](https://www.openvoiceos.com)
+- **Plasma Bigscreen**: An open UI stack for your own personal TV box using various open-source components like Plasma Bigscreen Launcher, Mycroft AI and Libcec based on KDE Neon Image. [Project Link](https://plasma-bigscreen.org/)
+- **Chatterbox**: The smart speaker anyone can build and teach, It's a smart speaker, like an Amazon Echo, but focused on education and privacy. The educational smart speaker to teach kids about coding, artificial intelligence and privacy. [Project Link](https://hellochatterbox.com/)
+- **Mycroft ROS**: This package provides capability to run the Text-To-Speech, Speech-To-Text and Skill functionality of Mycroft AI as ROS nodes providing topics, services and actions for communication. [Project Link](http://wiki.ros.org/mycroft_ros)
 
-# Wake word listeners and utilities
+### Mycroft Frameworks
+**_A list of frameworks built to extend Mycroft's capabilities beyond the default behavior which can be used standalone or in conjuction with each other_**
 
-# Skill management
+- **Lingua Franca**: Mycroft's multilingual text parsing and formatting library. A framework that is adopted as the common language between speakers with different native tongues. [Project Link](https://github.com/MycroftAI/lingua-franca)
```

Comment:
should this be removed? its an official mycroft lib, not community

Reply:
well, it has been community maintained so i guess it's OK. But in practice the community does not decide what happens there, a mycroft person can block or reject things at will